test/decomposer/test_mckay_decomposer.py

```diff
@@ -2,6 +2,7 @@
 
 import math
 
+import numpy as np
```

Comment:
Do we still need the `test_arbitrary` if we have the `test_full_sphere`?

Reply:
I will delete